test/DoctrineORMModuleTest/Console/Helper/MigrationsConfigurationHelperTest.php

```diff
@@ -0,0 +1,96 @@
+ * and is licensed under the MIT license.
+ */
+
+namespace DoctrineORMModuleTest\Console\Helper;
+
+use DoctrineModule\Component\Console\Input\RequestInput;
+use DoctrineORMModule\Console\Helper\MigrationsConfigurationHelper;
+use DoctrineORMModuleTest\ServiceManagerFactory;
+use Laminas\Console\Request;
+use Laminas\ServiceManager\Exception\ServiceNotFoundException;
+use Laminas\ServiceManager\ServiceManager;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Console\Input\InputDefinition;
+use Symfony\Component\Console\Input\InputOption;
+
+/**
+ * Tests for {@see \DoctrineORMModule\Service\MigrationsCommandFactory}
+ *
+ * @covers \DoctrineORMModule\Service\MigrationsCommandFactory
+ */
+class MigrationsConfigurationHelperTest extends TestCase
+{
+    /** @var ServiceManager */
+    private $serviceLocator;
+
+    public function setUp(): void
+    {
+        $this->serviceLocator = ServiceManagerFactory::getServiceManager();
+    }
+
+    public function testCreate(): void
+    {
+        $helper = new MigrationsConfigurationHelper($this->serviceLocator);
+
+        $this->assertInstanceOf(
+            MigrationsConfigurationHelper::class,
+            $helper
+        );
+    }
+
+    public function testGetDefaultMigrationConfig(): void
+    {
+        $inputOption     = new InputOption('object-manager', [], InputOption::VALUE_REQUIRED);
+        $inputDefinition = new InputDefinition([$inputOption]);
+        $request         = new Request([]);
+        $requestInput    = new RequestInput($request, $inputDefinition);
+        $helper          = new MigrationsConfigurationHelper($this->serviceLocator);
+
+        $configuration = $helper->getMigrationConfig($requestInput);
+
+        $this->assertEquals(
+            $this->serviceLocator->get('doctrine.migrations_configuration.orm_default'),
+            $configuration
+        );
+    }
+
+    /**
+     * This test queries for a non-orm_default object manager
+     * and throws an exception when the configuration is not found
+     */
+    public function testGetNonDefaultMigrationConfig(): void
+    {
+        $this->expectException(ServiceNotFoundException::class);
+        $this->expectExceptionMessage(
+            'Unable to resolve service '
+                . '"doctrine.migrations_configuration.orm_some_other_name" to '
+                . 'a factory; are you certain you provided it during configuration?'
+        );
+
+        $inputOption     = new InputOption('object-manager', [], InputOption::VALUE_REQUIRED);
+        $inputDefinition = new InputDefinition([$inputOption]);
+        $request         = new Request([
+            'index.php',
+            '--object-manager=doctrine.entitymanager.orm_some_other_name',
+        ]);
+        $requestInput    = new RequestInput($request, $inputDefinition);
+        $helper          = new MigrationsConfigurationHelper($this->serviceLocator);
+
+        $configuration = $helper->getMigrationConfig($requestInput);
```

Comment:
```suggestion
        (new MigrationsConfigurationHelper($this->serviceLocator))->getMigrationConfig($requestInput);
```

or alternatively remove the unused `$configuration` variable.

Reply:
Used suggested code which does not assign the helper to a variable.